### PR TITLE
Tie the globe icon button to publishing modal window

### LINKED ISSUE #881: Tie the globe icon button to the same action as 'Publish to Web'
main view

required popup

### PR DESCRIPTION
**Pull request by reggie7**
Fixes #881

### CODE REVIEW

**Review by reusr1 — APPROVED** [review]

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue [hunk 1]
```diff
@@ -247,6 +247,13 @@
             </div>
         </div>
     </div>
+
+      <admin-components-publishingmodal
+          v-if="publishDialogPath"
+          :isOpen="publishDialogPath"
+          :path="publishDialogPath"
+          @complete="closePublishing"
```
Comment (flx-sta) [inline admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue]: please move `@complete` below `:modalTItle`

(order is: first `v-bind/:` then `v-on/@`)

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue [hunk 2]
```diff
@@ -247,6 +247,13 @@
...
+          :modalTitle="`Web Publishing: ${publishDialogPath.split('/').pop()}`" />
```
Comment (flx-sta) [inline admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue]: What happens if `publishDialogPath` is `null/undefined`?
Is that even possible?

**Review by flx-sta — APPROVED** [review]
👀 apart from some code-styling stuff and a possible NPE issue: lgtm 🚀

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue (hunk 1 above)
Comment (reggie7) [reply admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue]: done

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue (hunk 2 above)
Comment (reggie7) [reply admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue]: Nothing thanks to `v-if` above. And yes - we explicitly set it to `null` in `closePublishing` below.

Comment (flx-sta) [reply admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue]: man im blind..

**Pull request merged**